spec/models/update_subscriber_list_spec.rb

```diff
@@ -0,0 +1,119 @@
+require "spec_helper"
+
+RSpec.describe UpdateSubscriberList do
+  let(:content_id) { SecureRandom.uuid }
+  let(:document_title) { "Example Title" }
+  let(:major_change_document) do
+    {
+      "base_path" => "/foo",
+      "content_id" => content_id,
+      "title" => document_title,
+      "update_type" => "major",
+    }
+  end
+
+  let(:minor_change_document) do
+    {
+      "base_path" => "/foo",
+      "content_id" => content_id,
+      "title" => document_title,
+      "update_type" => "minor",
+    }
+  end
```

Comment:
Rather than have two almost identical definitions, just do `let(:document)` and have the `update_type` be another `let`

Reply:
Yeah that's a good shout, these used to be more varied but I forgot we paired these down in one of the last commits.